src/textual/_xterm_parser.py

```diff
@@ -50,7 +50,7 @@
 
 
 class XTermParser(Parser[Message]):
-    _re_sgr_mouse = re.compile(r"\x1b\[<(-?\d+);(-?\d+);(-?\d+)([Mm])")
+    _re_sgr_mouse = re.compile(r"\x1b\[<(\d+);(-?\d+);(-?\d+)([Mm])")
```

Comment:
The irony is that the terminal shouldn't send hyphens in any of those fields. But I guess we haven't seen negative mouse buttons in the wild, so this makes sense.